File: Assets/Scripts/Vehicle/VehicleStats.cs
using UnityEngine;

[System.Serializable]
public class VehicleStats
{
    [Header("Movement")]
    public float maxSpeed = 28f;
    public float acceleration = 20f;
    public float brakePower = 30f;
    public float reverseSpeedFactor = 0.45f;

    [Header("Handling")]
    public float handling = 95f;
    public float lateralGrip = 9f;
    public float driftGrip = 3.5f;

    [Header("Drift")]
    public float driftMinSpeed = 8f;
    public float driftChargeRate = 0.8f;
    public float driftBoostPower = 10f;

    [Header("Combat")]
    public float hitRecovery = 1.4f;
}

File: web/src/vehicle/VehicleStats.js
export class VehicleStats {
  constructor({
    name = "Racer",
    maxSpeed = 365,
    acceleration = 260,
    brake = 320,
    reverseFactor = 0.45,
    handling = 2.4,
    driftHandling = 3.1,
    grip = 9,
    driftGrip = 3.2,
    driftChargeRate = 0.95,
    driftMinSpeed = 120,
    driftBoostScale = 150,
  } = {}) {
    this.name = name;
    this.maxSpeed = maxSpeed;
    this.acceleration = acceleration;
    this.brake = brake;
    this.reverseFactor = reverseFactor;
    this.handling = handling;
    this.driftHandling = driftHandling;
    this.grip = grip;
    this.driftGrip = driftGrip;
    this.driftChargeRate = driftChargeRate;
    this.driftMinSpeed = driftMinSpeed;
    this.driftBoostScale = driftBoostScale;
  }
}
